schnorr_fun/src/frost.rs

```diff
@@ -437,27 +470,41 @@ impl<H: Digest<OutputSize = U32> + Clone, NG: NonceGen> Frost<H, NG> {
     /// Create our secret shares and proof-of-possession to be shared with other participants.
     ///
     /// Each secret share needs to be securely communicated to the intended participant.
-    /// The proof-of-possession should be sent to all participants.
     ///
-    /// ## Return value
+    /// # Return value
     ///
-    /// Returns a vector of secret shares where the index represents the signer index and a
-    /// proof-of-possession as a `Signature`.
-    pub fn create_shares(
+    /// Returns a vector of secret shares where the index represents the signer index,
+    /// and a proof of possession.
+    pub fn create_shares_and_pop(
```

Comment:
This function doesn't make sense anymore because you need to pass the indexes you want to generate a share for. Return a `BTreeMap` of `PartyIndex` to share?